universal-application-tool-0.0.1/app/export/Exporter.java

```diff
@@ -0,0 +1,8 @@
+package export;
+
+import java.io.Writer;
+import models.Applicant;
+
+public interface Exporter {
```

Comment:
(blocking) can you add comments to the interface, so its clear what is going on?

Reply:
Sure.